src/po/sr.po

```diff
@@ -2457,29 +2185,115 @@ msgstr "E324: PostScript излазни фајл не може да се отв
 msgid "E456: Can't open file \"%s\""
 msgstr "E456: Фајл \"%s\" не може да се отвори"
 
-msgid "E456: Can't find PostScript resource file \"prolog.ps\""
-msgstr "E456: PostScript resource фајл \"prolog.ps\" не може да се пронађе"
+msgid "E456: Can't find PostScript resource file \"prolog.ps\""
+msgstr "E456: PostScript resource фајл \"prolog.ps\" не може да се пронађе"
+
+msgid "E456: Can't find PostScript resource file \"cidfont.ps\""
+msgstr "E456: PostScript resource фајл \"cidfont.ps\" не може да се пронађе"
```

Comment:
Why leave "resource" instead of using "resurs"? Again "nije pronadjen" instead of the self-referential "ne moze da se pronadje".

Reply:
Of course, resurs is the right word. I can't remember why have I left it like this